/**:
  ros__parameters:
    H: 1000
    beta_degrees: 54.73
    function_names:
      pseudo_inverse: pseudoInvFunc
      height_function: hFunc
    code_generation:
      enable_cpp: true
      output_file: "L_p_func.cpp"

attitude_dynamics_node:
  ros__parameters:
    dynamics:
      J:
        xx: 280e6
        yy: 140e6
        zz: 420e6
      mu: 3.986e14
      r_orbit:  7e6

    timing:
      torque_dt:      0.1
      pub_dt:         0.1
      publish_every:  10

    initial:
      attitude: [0.0, 0.0, 0.0, 1.0]
      angvel:   [0.0, 0.0, 0.0]
      angacc:   [0.0, 0.0, 0.0]
